packages/webview_flutter/webview_flutter_wkwebview/test/src/web_kit_webview_widget_test.dart

```diff
@@ -84,6 +89,26 @@ void main() {
       await buildWidget(tester);
     });
 
+    testWidgets('WKUIDelegate.onCreateWebView loadRequest',
```

Comment:
Can we name this something more descriptive of the behavior it's testing, like 'requests to open a new window instead open inline'?